softwareconfiguration/autofe-reldata/src/main/java/autofe/db/search/NoSolutionFromRandomCompletionException.java

```diff
@@ -0,0 +1,22 @@
+package autofe.db.search;
+
+public class NoSolutionFromRandomCompletionException extends RuntimeException {
+

```

Comment:
Should be a checked Exception I suppose

Reply:
Resolved, now it is a checked exception.